src/lib/components/dropdown-menu.svelte

```diff
@@ -41,11 +41,12 @@
   });
 </script>
 
-<div class="relative inline mx-2" bind:this={menu}>
+<div class="relative inline mx-2" bind:this={menu} data-cy={$$props.dataCy}>
```

Comment:
You don't need to change this now, but you can also do `$$props['data-cy']` if you want to stay consistent with the resulting HTML.

Reply:
i've tried `$$props['data-cy']` at first and this resulted in an undefined